app/scripts/modules/amazon/src/serverGroup/serverGroup.transformer.ts

```diff
@@ -0,0 +1,102 @@
+import { module, IPromise } from 'angular';
+import { defaults } from 'lodash';
+
+import { IVpc } from '@spinnaker/core';
+
+import { IAmazonScalingAdjustment, IAmazonScalingPolicy, IAmazonServerGroup, IAmazonStepAdjustment } from '../domain';
+import { VPC_READ_SERVICE, VpcReader } from '../vpc/vpc.read.service';
+
+export class AwsServerGroupTransformer {
+  public constructor(private vpcReader: VpcReader) { 'ngInject'; }
+
+  private addComparator(alarm: any): void {
```

Comment:
Does `alarm` have a known shape? (even just declaring an interface with `comparisonOperator` and `comparator` would be good, and we could build it out more as needed)